content/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services.md

```diff
@@ -10,59 +10,65 @@ toc: true
 weight: 400
 ---
 
-This guide explains how to use the NGINX Open Source or F5 NGINX Plus Ingress Controller for Kubernetes as the Ingress controller for a Kubernetes cluster hosted in the Amazon Elastic Kubernetes Service (EKS).
+This guide explains how to use NGINX Open Source or F5 NGINX Plus Ingress Controller for Amazon Elastic Kubernetes Services (EKS).
```

Comment:
```suggestion
This guide explains how to use NGINX or F5 NGINX Plus with NGINX Ingress Controller for Amazon Elastic Kubernetes Services (EKS).
```

The product is named "NGINX Ingress Controller", prefixed with F5 if it's the first commercial product named on a page. We generally don't specify "Open Source" for the base software, since it's the "default" - the difference is indicated with the "Plus" suffix.